.github/workflows/cd.yaml

```
@@ -47,9 +47,9 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 8
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3.0.0
```

Use major version only to be consistent with Jenkins core and to follow recommendations from Jesse Glick and others.
```suggestion
        uses: actions/setup-java@v3
```